### ISSUE TITLE
tsserver is deprecated, use ts_ls instead

### ISSUE THREAD

**Issue by skinner12** [nvim-lua/kickstart.nvim#1128]
## Describe the bug

tsserver is deprecated, use ts_ls instead.
Feature will be removed in lspconfig 0.2.0.

## To Reproduce

Update to latest `nvim-lspconfig` and delete comment from this line: https://github.com/nvim-lua/kickstart.nvim/blob/master/init.lua#L618

## Neovim Version

NVIM v0.10.1
Build type: Release
LuaJIT 2.1.1723675123

```
```

**Comment by drio** [comment]
I fixed this by using [this](https://github.com/neovim/nvim-lspconfig/pull/3232#issuecomment-2331025714) from [ic-768](https://github.com/ic-768). 

````diff
diff --git a/init.lua b/init.lua
index 832d8d4..3efcb99 100644
--- a/init.lua
+++ b/init.lua
@@ -667,6 +667,10 @@ require('lazy').setup({
             -- This handles overriding only values explicitly passed
             -- by the server configuration above. Useful when disabling
             -- certain features of an LSP (for example, turning off formatting for tsserver)
+            -- https://github.com/neovim/nvim-lspconfig/pull/3232
+            if server_name == 'tsserver' then
+              server_name = 'ts_ls'
+            end
             server.capabilities = vim.tbl_deep_extend('force', {}, capabilities, server.capabilities or {})
             require('lspconfig')[server_name].setup(server)
           end,
```

**Comment by Cheveniko** [comment]
Hey I opened a PR to fix this! #1131

**Comment by feoh** [comment]
PR merged. Closing.
